docs/api-reference/loaders-storage-targets/databricks-streaming-loader/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Databricks Streaming Loader"
+sidebar_label: "Databricks Streaming Loader"
+sidebar_position: 5
+---
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+import LoaderDiagram from '@site/docs/api-reference/loaders-storage-targets/databricks-streaming-loader/_diagram.md';
+import DeployOverview from '@site/docs/api-reference/loaders-storage-targets/databricks-streaming-loader/_deploy-overview.md';
+```
+
+## Overview
+
+:::info Early release
+
+This is an early release. It relies on Databricks features that are in preview as of August 2025, e.g. [File Events](https://docs.databricks.com/aws/en/ingestion/cloud-object-storage/auto-loader/file-notification-mode#file-events).
+
+Also, you will need a premium Databricks plan to use Lakeflow Declarative Pipelines.
+
+:::
+
```

Comment:
please can you bring the first sentence of `_diagram.md` out of the reusable block so it's the intro sentence to this page? feels weird to go straight into tabs